cypress/e2e/tests/testSurvey.cy.ts

```diff
@@ -0,0 +1,83 @@
+import * as assignSurvey from "../helpers/adminHelpers/assignSurvey";
+import * as homeHelpers from "../helpers/homeHelpers";
+import * as answerSurvey from "../helpers/answerSurvey";
+import moment from "moment";
+import "moment/min/locales";
+
+describe("Create new account page", () => {
+  beforeEach(() => {
+    assignSurvey.assignSurveyToPatient();
+  });
+
+  it("FDHA-3992 Patient should be able to answer SER-ISD-001 - Body Weight survey", () => {
+    cy.loginWebPatient(
+      Cypress.env("emailWebPatient"),
+      Cypress.env("passwordWebPatient")
+    );
+    cy.wait(5000);
+    cy.get(homeHelpers.BUTTON_SURVEY_CARD("Surveys"))
+      .should("be.visible")
+      .click();
+    cy.contains("h6", "SER-ISD-001 - Body Weight");
+    cy.contains("h6", "OPEN");
+
+    cy.get('[data-testid="SURVEY_CARD_SER-ISD-001 - Body Weight"]').click();
+
+    cy.get(answerSurvey.BUTTON_CLOSE).should("contain", "Close");
+
+    cy.contains("h4", "SER-ISD-001 - Body Weight");
+    cy.get(answerSurvey.TEXT_DATE)
+      .invoke("text")
+      .then((text) => {
+        const formattedDate = moment(text, "dddd, MMMM DD", "en", true);
+        expect(formattedDate.isValid()).to.be.true;
+        const formattedDateText = formattedDate.format("dddd, MMMM DD");
+        expect(formattedDateText).to.equal(text);
+        cy.get(answerSurvey.TEXT_DATE).should("contain", formattedDateText);
+      });
+
+    cy.contains("What is your weight today?");
+    cy.contains("Please write the full number including the decimal.");
+    cy.get(answerSurvey.INPUT_VALUE_LBS).should(
+      "have.attr",
+      "placeholder",
+      "000.0"
+    );
```

Comment:
please move everything that is checking the interface to a function like `verifyWeightScreenContent`()